src/main/java/cz/muni/fi/ComponentType.java

```diff
@@ -0,0 +1,8 @@
+package cz.muni.fi;
+
+/**
+ * Author: Richard Hrmo
+ */
+public enum ComponentType {
```

Comment:
prosim refaktor mena na EComponentType

Reply:
Nechápem na čo tam ma byť to E, chápem že to je enum, ale aj tak ... ok

Reply:
Konvencia, neviem ci len u nas v praci alebo celkovo, zisti a ked mi daj vediet lebo ja svoj enum tak pomenovany mam

Reply:
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html